config/locales/en/helpers.yml

```diff
@@ -272,9 +272,9 @@ en:
           years: Years
           no_length: No length was given
       steps_conviction_conviction_length_form:
-        weeks: Number of weeks
-        months: Number of months
-        years: Number of years
+          weeks: Number of weeks
```

Comment:
There seems to be one level deeper than necessary (one less tab here)

Reply:
good spot!